[2025-01-24T20:00:16.943+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-24T20:00:16.964+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task manual__2025-01-24T20:00:00.044749+00:00 [queued]>
[2025-01-24T20:00:16.977+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pull_gazettes_data.init_folders_task manual__2025-01-24T20:00:00.044749+00:00 [queued]>
[2025-01-24T20:00:16.978+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-01-24T20:00:16.995+0000] {taskinstance.py:2888} INFO - Executing <Task(InitFoldersTask): init_folders_task> on 2025-01-24 20:00:00.044749+00:00
[2025-01-24T20:00:17.024+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=417) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-24T20:00:17.028+0000] {standard_task_runner.py:72} INFO - Started process 494 to run task
[2025-01-24T20:00:17.027+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pull_gazettes_data', 'init_folders_task', 'manual__2025-01-24T20:00:00.044749+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmpxxo6ymot']
[2025-01-24T20:00:17.032+0000] {standard_task_runner.py:105} INFO - Job 4: Subtask init_folders_task
[2025-01-24T20:00:17.106+0000] {task_command.py:467} INFO - Running <TaskInstance: pull_gazettes_data.init_folders_task manual__2025-01-24T20:00:00.044749+00:00 [running]> on host 972b45cbfb12
[2025-01-24T20:00:17.223+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pull_gazettes_data' AIRFLOW_CTX_TASK_ID='init_folders_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-24T20:00:00.044749+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-24T20:00:00.044749+00:00'
[2025-01-24T20:00:17.226+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-24T20:00:17.298+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-24T20:00:17.299+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=pull_gazettes_data, task_id=init_folders_task, run_id=manual__2025-01-24T20:00:00.044749+00:00, execution_date=20250124T200000, start_date=20250124T200016, end_date=20250124T200017
[2025-01-24T20:00:17.365+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-24T20:00:17.396+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-24T20:00:17.399+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
